packages/storage-plus/src/de.rs

```diff
@@ -106,7 +104,6 @@ macro_rules! integer_de {
             #[inline(always)]
             fn from_vec(value: Vec<u8>) -> StdResult<Self::Output> {
                 Ok(<$t>::from_be_bytes(value.as_slice().try_into()
-                    // FIXME: Add and use StdError try-from error From helper
                     .map_err(|err: TryFromSliceError| StdError::generic_err(err.to_string()))?))
```

Comment:
Arg. Here you can't do the same

```
.map_err(StdError::generic_err)?
```

because the function takes `Into<String>` instead of `ToString`?

Reply:
✔️